docs/docs/overview/quickstart/get-started-with-vanilla-js.md

```diff
@@ -0,0 +1,315 @@
+---
+sidebar_position: 8
+sidebar_label: Get started with vanilla JS
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Vanilla JS Quickstart
+
+Learn how to integrate Novu into your vanilla JS app on the fly. Send notifications across channels (SMS, Email, Chat, Push) with the help of a rich, customizable, real-time In-App notification center
+
+## Requirements
+
+To follow the steps in this quickstart, you'll need:
+
+- A Novu account. [Sign up for free](http://web.novu.co) if you don’t have one yet.
+- A working vanilla JS development environment.
+
+You can find the code for this project [here](https://github.com/novuhq/vanillajs-quickstart)
+
+## Loading Notification center component from the CDN
+
+The Notification Center Web Component is a custom element that can be used in any web application. It is built by wrapping the React components and consists from its core the three pieces: NovuProvider, PopoverNotificationCenter, and NotificationBell combined together into one.
```

Comment:
Please remove `It is built by wrapping the React components and consists from its core the three pieces: NovuProvider, PopoverNotificationCenter, and NotificationBell combined together into one.` .  That piece of info is not really needed here.

It consists